src/js/common/datetime.js

```diff
@@ -610,8 +610,11 @@ datetime = {
     convertStartDayToLastDay: function(d) {
         var date = new TZDate(d);
         var isStartOfDay = datetime.isStartOfDay(d);
+        if (isStartOfDay) {
+            date.setDate(date.getDate() - 1);
+        }
 
-        return isStartOfDay ? date.setDate(date.getDate() - 1) : date;
```

Comment:
When writing code for the first time
It was my mistake.
It was a problem by returning the number value when calling `setDate`.